pkg/util/mount/nsenter_mount.go

```diff
@@ -249,7 +249,7 @@ func (mounter *NsenterMounter) GetFileType(pathname string) (FileType, error) {
 		return FileTypeBlockDev, nil
 	case "directory":
 		return FileTypeDirectory, nil
-	case "regular file":
+	case "regular file", "regular empty file":
```

Comment:
is it possible to unit test nsenter?

Reply:
It will be difficult.

Reply:
Can there be e2e tests then?  E2E tests should be able to run against any Kubernetes cluster.

Reply:
Our official CI does not run containerized kubelet, which means `nsenter` mounter will not be used.

Reply:
But at least there would be reproducible tests available that someone can run against a containerized kubelet cluster, instead of the current state of manual testing.  As a sig, we are trying to find an owner to enable automated ci against containerized kubelet.  We are seeing too many bugs lately in this area and we don't have an easy way to validate the bug fixes.

Reply:
To clarify, I'm not asking that you be owner of enabling the CI infra.  I'm just trying to say that writing e2e tests will not be a wasted effort. We'll get consistent test cases across standard and containerized environments, and eventually it will be automated for the latter too.

Reply:
> writing e2e tests will not be a wasted effort. We'll get consistent test cases across standard and containerized environments, and eventually it will be automated for the latter too

Right. Agree. Will try to find a way to test this.